content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -153,6 +153,59 @@ Allowed customization are:
 Please note that the configuration field `kubernetesVersion` or the command line flag
 `--kubernetes-version` affect the version of the images.
 
+### Installing custom addons (Alpha) {#addon-installer}
+
+Kubeadm will normally install CoreDNS/kube-dns and kube-proxy using manifests built into the kubeadm binary.
+Using the `AddonInstaller` feature-gate (alpha), you can instruct kubeadm to install addons from other sources.
+Kubeadm will no longer use the in-binary manifests when using the `AddonInstaller` feature-gate.
+
+This feature is driven by a Component Config called the `addons.config.x-k8s.io/AddonInstallerConfiguration`.
+The `AddonInstallerConfiguration` can be specified in the kubeadm config-file.
+If the `AddonInstaller` feature-gate is specified, but a configuration is not supplied, kubeadm will default the API. It is planned to default to installing version appropriate CoreDNS+kube-proxy, but these addons are not yet implemented.
```

Comment:
I recommend:
```suggestion
If you enable the `AddonInstaller` feature gate, but you don't specify a configuration, kubeadm uses its own default configuration.
```
([avoid making statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future))